sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/AccountMetadataKey.kt

```diff
@@ -0,0 +1,62 @@
+package cash.z.ecc.android.sdk.model
+
+import cash.z.ecc.android.sdk.internal.model.JniMetadataKey
+import cash.z.ecc.android.sdk.tool.DerivationTool
+
+/**
+ * A [ZIP 325](https://zips.z.cash/zip-0325) Account Metadata Key.
```

Comment:
Should this page exist already, or is it a pre-existing ZIP? I get 404 on this URL.

Reply:
That URL will exist once https://github.com/zcash/zips/pull/979 is finished and merged.